internal/confmapprovider/discovery/properties/property.go

```diff
@@ -222,7 +222,7 @@ func unwordify(text string) (string, error) {
 		for len(decoded) < 4 {
 			decoded = append([]byte{0}, decoded...)
 		}
-		r := int32(binary.BigEndian.Uint32(decoded))
+		r := binary.BigEndian.Uint32(decoded)
```

Comment:
Internally `"%c"` casts everything to `uint64` and doesn't care for signal, so it is safe to remove the cast to `int32`.